### PR TITLE
Add build script for stdlib test projects

### PR DESCRIPTION
**Pull request by nfurfaro**
This is a bash script that @simonr0204 wrote to simplify building multiple test projects and dependencies.

Ref: (Old PR in `sway-lib-std` repo) https://github.com/FuelLabs/sway-lib-std/pull/100

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Place in root of project and run to build the project and all its tests and artifacts
+FILES="./tests/test_*/*"
```
Comment (adlerjohn) [inline build.sh:4]: This should be changed to just look under sway-lib-std and sway-lib core

**Review by adlerjohn — CHANGES_REQUESTED** [review]

File: build.sh (hunk 1 above)
Comment (nfurfaro) [reply build.sh:4]: It already only looks under `sway-lib-std` as the script is in the root of that sub-folder.
What would we want it to build under `sway-lib-core`?

Comment (adlerjohn) [reply build.sh:4]: Wait no, the script is in the root of the sway repo!

Comment (nfurfaro) [reply build.sh:4]: Both `sway-lib-std` & `sway-lib-core` get built for each test project, so I don't think it's necessary to add another step to the script to build `sway-lib-core`.

**Review by adlerjohn — APPROVED** [review]
utACK

**Pull request merged**